### DISCUSSION TITLE
The complete set of proofs for the invalidity of the special theory of relativity

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [a-cosmology-group/acg#272] [Cosmology Models]
https://www.researchgate.net/publication/381755574_The_complete_set_of_proofs_for_the_invalidity_of_the_special_theory_of_relativity
https://arxiv.org/abs/1411.3987

**Comment by LifeIsStrange** [comment]
@RedshiftDrift did I serendipitously stumble upon your grandfather's papers? :)
https://www.newtonphysics.on.ca/illusion/

**Reply by ExpEarth** [reply]
That's Marmet père, not grand-père!  I corresponded with Paul years ago before knowing Louis.  Paul Marmet was at least as ATM as Louis is.

**Reply by RedshiftDrift** [reply]
"ATM"??

**Reply by ExpEarth** [reply]
Yes, for "against the mainstream".  That was a common term in discussion groups back then, maybe still.  Certainly fits you and your dad!

**Comment by Francois-Zinserling** [comment]
> https://www.researchgate.net/publication/381755574_The_complete_set_of_proofs_for_the_invalidity_of_the_special_theory_of_relativity https://arxiv.org/abs/1411.3987

TLDR
Do the good guys win in the end?

**Reply by LifeIsStrange** [reply]
Well I have not read this paper enough to give a proper TL;DR, however I am familiar with the experiments they cite, and indeed it seems that the one way speed of light is variable and there is empirical evidence for an aether or Lesagian graviton background.
Given our shared interest in lesagian gravity, I would like if you could analyse the following things:
1) Give a lesagian interpretation of the [Allais effect](https://en.wikipedia.org/wiki/Allais_effect) and how semantically different is it from https://en.wikipedia.org/wiki/Gravitational_shielding
2) Direct empirical evidence for aether drag in Jupiter's moon IO
https://www.scirp.org/journal/paperinformation?paperid=115246
3) gravitational anomaly in the MM experiment
https://rdcu.be/dQYyA
> Interaction of the gas in the optical pathways of an interferometer with a field of moving particles (gravitons?) might account for the pulsatile characteristics of signals in M–M experiments

such signal has been reproduced in distinct gravitational tests
4) gravitational shielding/artificial weight reduction has been observed in the lab and reproduced, in superconductors confined in a magnetic field
> https://www.sciencedirect.com/science/article/abs/pii/092145349290055H

related but more niche
https://www.semanticscholar.org/paper/Spinning-Electric-Field-As-a-Virtual-Gyroscope%3A-On-Krinker/fa843b276cdfc72ccca99584ab68f9c686b22725
https://www.semanticscholar.org/paper/The-phenomenon-of-weight-reduction-of-a-spinning-Wayte/b403ab1bdd10af5b2923fc14a6d247d974bcf506
https://www.semanticscholar.org/paper/Characterization-of-Anomalous-Forces-in-Dielectric-Porcelli-Filho/1c3374d78b23f137da2ac0a3114e31b122f273db

**Reply by Francois-Zinserling** [reply]
It's going to be difficult to ascribe a gravitation anomaly to push-gravity(PG), since every anomaly has been analysed to death and given 'some' mainstream explanation. Some are so loaded with math they are near-impossible to unravel. 
Unfortunately the Allais effect isn't reliably reproducable which makes it difficult to really understand what was going on there.

Shielding is a real and observed effect in electrostatics and electrodynamics, which is why it appeared natural to explore for a similar phenomenon in gravity. It is there, but we do not yet recognise it for what it is.

Once the 'gravity attracts' idea is shirked, it becomes evident that even tides are due to gravitational shielding in a PG scenario. Then shielding is no longer a problem to be avoided, but rather a new field of exploration to be embraced.
With a vision of aether IN > aether OUT causing matter to be pushed together, it's easier to understand how e.g. the presence of the moon can perturb the IN component (for earth) on a specific vector, causing e.g. less push down on the ocean, and making it appear as if the moon is attracting the water upward. 

An interesting note in the wiki gravitational shielding page, where Marjorana predicts a loss of mass when 2 masses approach. This is NB and in my opinion is what causes (apparent) loss-of-mass in nuclear and even atomic bonds. Since the aether determines inertia, hence mass, a weaker aether between 2 masses means some 'mass is lost'. This is apparent and not real, since no actual matter has been lost in the process. The energy release comes from the aether.

If atoms are mostly empty spaces but nuclei are not, then the strong force is also PG, but with no (or less) empty spaces between nuclei, the PG force is at its maximum on that scale. There is also evidence that bound nuclei don't just 'lose mass', but mass is only lost along the axis or plane along which the nuclei are bound. This too supports Marjorana's idea but since his was associated with gravity it was outright rejected at the time.

The 'mass is lost' idea is not limited to atoms, but also answers to galactic rotation, especially observed for a disk, where maximum PG is from the edge of the disk inward to the centre but gets gradually weaker into the plane of the disk. Hence higher rotation on the perimeter, or better seen as maximum rotation on the perimeter and weaker rotation further in.

These are but some of the spin-offs which may or may not add support to PG.

I have not read the detail of all the links you provided, but in my opinion a manipulation of photon pairs should make at least a partial anti-gravity, or increase or reduction in gravity, possible. The Rancourt experiment may give us a clue.
[Rancourt on ResearchGate](https://www.researchgate.net/publication/353090350_Simple_experiment_to_demonstrate_that_light_acts_on_gravity)

**Reply by HanDeBruijn** [reply]
@LifeIsStrange and @Francois-Zinserling .

Maybe of interest to both of you. I've once met the following references on the internet.
- [Gravity and Antigravity](https://www.davidpratt.info/gravity.htm) part I
- [Gravity and Antigravity](https://www.davidpratt.info/gravity2.htm) part II

**Reply by HanDeBruijn** [reply]
Now I see that this author - David Pratt - has written an incredible [**whole lot**](https://www.davidpratt.info/homepage.htm) more, about a huge variety of subjects !!

- Very much **on topic**: [Space, Time and Relativity](https://www.davidpratt.info/relativ.htm) .
- Also interesting: [The Farce of Modern Physics](https://www.davidpratt.info/farce.htm). Eric Lerner is mentioned a few times.
- With links to [ACG](https://cosmology.info/) and Apeiron: [Trends in Cosmology: Beyond the Big Bang](https://www.davidpratt.info/cosmo.htm) . Keep searching and find
- [Black holes/big bang: a debate](https://www.davidpratt.info/bol.htm) , [Big Bang, Black Holes and Common Sense](https://www.davidpratt.info/bang.htm) , [Exploding the Big Bang](https://www.davidpratt.info/explode.htm) , ...

>Do the good guys win in the end?

**Comment by sahil5d** [comment]
> Albert Einstein and Leopold Infeld wrote in The Evolution of Physics (1938): "Can we formulate physical laws so that they are valid for all CS [coordinate systems], not only those moving uniformly, but also those moving quite arbitrarily, relative to each other? If this can be done, our difficulties will be over. We shall then be able to apply the laws of nature to any CS. The struggle, so violent in the early days of science, between the views of Ptolemy and Copernicus would then be quite meaningless. Either CS could be used with equal justification. The two sentences, 'the sun is at rest and the Earth moves', or 'the sun moves and the Earth is at rest', would simply mean two different conventions concerning two different CS. Could we build a real relativistic physics valid in all CS; a physics in which there would be no place for absolute, but only for relative, motion? This is indeed possible!" 
> Despite giving more respectability to the geocentric view than Newtonian physics does, relativity is not geocentric. Rather, relativity states that the Sun, the Earth, the Moon, Jupiter, or any other point for that matter could be chosen as a center of the Solar System with equal validity. 
> https://en.wikipedia.org/wiki/Geocentric_model#Relativity

SR makes as much sense as believing the Solar System revolves around Uranus.

**Reply by HanDeBruijn** [reply]
>SR makes as much sense as believing the Solar System revolves around Uranus.

It cannot be repeated enough, but we have been going through this in [thread 215](https://github.com/orgs/a-cosmology-group/discussions/215#discussioncomment-9472304), haven't we?
